cmd/convox/templates/init/python/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "2"
+services:
+  web:
+    build: .
+    labels:
+      - convox.port.443.protocol=tls
+      - convox.port.443.proxy=true
```

Comment:
This line enables PROXY protocol. Will the gunicorn backend be able to handle that? We might need to run something in front. In our rails image, for example we use nginx to read the PROXY protocol headers and set HTTP headers for the app backend:

https://github.com/convox/rails/blob/master/conf/nginx.conf#L35-L38
https://github.com/convox/rails/blob/master/bin/web#L6-L7